$MirOS$

	Never use the accent gravis where an apostrophe or
	typographic quotation mark was expected; suggested
	for upstream inclusion.

--- src/chrtrans/def7_uni.tbl.orig	Mon Feb 13 02:45:53 2012
+++ src/chrtrans/def7_uni.tbl	Sat Mar  3 18:20:22 2012
@@ -1356,8 +1356,7 @@ U+200a:
 U+2014 "--"
 U+2016:||
 U+2017:=2
-0x60	U+2018		# left single quotation mark  <`>
-0x27	U+2019-U+201b	# various single quotation marks <'>
+0x27	U+2018-U+201b	# various single quotation marks <'>
 0x22	U+201c-U+201f	# various double quotation marks <">
 U+2020:/-
 U+2021:/=
